classes/booking_manager.php

```diff
@@ -53,6 +53,9 @@ class booking_manager {
     /** @var bool Whether or not the bookings are loaded from a file. */
     private $usefile = true;
 
+    /** @var bool Wether the confirmation emails should not be send */
```

Comment:
```suggestion
    /** @var bool Whether the confirmation emails should not be sent */
```
OR (to avoid mental gymnastics with double negatives)
```suggestion
    /** @var bool When true, confirmation emails will not be sent */
```